src/common/types/roomRequest.ts

```
@@ -402,8 +403,11 @@ export const roomRequestSchema = roomRequestDataSchema.
 
 export type RoomRequestFormValues = z.infer<typeof roomRequestSchema>;
 
+export const roomRequestCompatShim = {
+  requestsSccsRoom: z.boolean().optional()
```

ESLint

[error] Insert `,`

(prettier/prettier)